pkg/build-deb.sh

```
@@ -0,0 +1,58 @@
+  sudo apt-get build-dep -y .
+}
+
+build() {
+
+  cd "$BUILD_DIR/snowflake-${SNOWFLAKE_VERSION}"
+  echo "Building Debian package..."
+  DISTRO=$(lsb_release -cs)
+  # SNOWFLAKE_DEB_VERSION carries the '~<pretag>' form for pre-releases so they
+  # sort below stable; it equals SNOWFLAKE_VERSION for a GA build.
+  rm -rf debian/changelog
+  echo "pgedge-snowflake (${SNOWFLAKE_DEB_VERSION}-${SNOWFLAKE_BUILDNUM}.${DISTRO}) unstable; urgency=low" >> debian/changelog
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)